front/src/modules/pathfinding/components/IncompatibleConstraints/IncompatibleConstraintsList.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { type FC, useMemo } from 'react';
+
+import { debounce, sortBy } from 'lodash';
+import { useMap } from 'react-map-gl/maplibre';
+
+import IncompatibleConstraintItem from './IncompatibleConstraintsItem';
+import type { IncompatibleConstraintItemEnhanced } from './type';
+
+interface IncompatibleConstraintListProps {
+  data: Array<IncompatibleConstraintItemEnhanced>;
+  hovered: Set<string>;
+  selected: Set<string>;
+  onHover: (itemId: string | null) => void;
+  onSelect: (itemId: string) => void;
```

Comment:
To improve readability

```suggestion
  formattedConstraints: IncompatibleConstraintItemEnhanced[];
  hoveredConstraintsIds: Set<string>;
  selectedConstraintsIds: Set<string>;
  onConstraintHover: (constraintId: string | null) => void;
  onConstraintSelect: (constraintId: string) => void;
```

Reply:
Adding `constraints` everywhere for me it's no-sense. We are already on a folder with this name, with a component with it, ... 
On a button on react you don't have `onButtonClick` just `onClick`. The component name give you the context.
IMHO, it doesn't improve the readability but instead spams my vision with `Contrainst` everywhere.